modules/redis/variables.tf

```diff
@@ -17,4 +17,14 @@ variable "network" {
 variable "memory_size_gb" {
   description = "The amount of memory which will be allocated to the Redis instance; this value must be expressed in gibibytes."
   type        = number
+}
+
+variable "reserved_ip_range" {
+  description = "This defines the IP range for REDIS peering connection"
+  type        = string
+}
```

Comment:
Do we want to set the default here?

Reply:
Ahh, I see, we are defaulting it elsewhere. I think we should keep the defaults incase someone is using the submodule outside of our main.tf.